wurst/event/UniqueDamageListening.wurst

```diff
@@ -0,0 +1,135 @@
+package UniqueDamageListening
```

Comment:
This package name is confusing/unclear. Maybe just merge this stuff into the main file? This will probably be a more common use case than the global modifiers.

Reply:
This was taken out of it just because it took a hell of a lot space in there and basically it's just a simple snippet no one is forced to use.

So you can make your own awesome performance optimization suited right for your needs out of this.

Reply:
This was taken out of it just because it took a hell of a lot space in there and basically it's just a simple snippet no one is forced to use.

So you can make your own awesome performance optimization suited right for your needs out of this.